ReportUtilityBills/measure.rb

```diff
@@ -294,7 +311,7 @@ def run(runner, user_arguments)
       utility_rates, utility_bills = setup_utility_outputs()
 
       # Get utility rates
-      warnings = get_utility_rates(hpxml_path, fuels, utility_rates, utility_bill_scenario, @hpxml_bldg.pv_systems)
+      warnings = get_utility_rates(hpxml_path, fuels, utility_rates, utility_bill_scenario, pv_systems, num_units)
```

Comment:
Rather than passing in a large array of PV objects, just pass in the one single value that you need (total building kW).

Reply:
Requires refactoring our unit test file a bit, too...